.github/ISSUE_TEMPLATE/socks-access-request.yml

```diff
@@ -0,0 +1,50 @@
+name: WIP SOCKS Access Request
+description: To request access to SOCKS
+title: Access for [individual]
+labels: ['external-request', 'operations', 'ops-access-request']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        1. Change the **Title** to include your name
+        2. Add the label used by your team (eg: `BAH-526`)
+        3. Do not remove `operations` label or the `ops-access-request` label; this will notify the VSP Operations team of your request
+        4. When the issue is closed you will be notified and can continue with on-boarding setup
+  - type: input
+    id: requestor-name
+    attributes:
+      label: Your Name
+      description: Please provide your name
+      placeholder: e.g. Jane Doe
+    validations:
+      required: true
+  - type: input
+    id: email
+    attributes:
+      label: Your Email
+      description: Please provide your work email
+      placeholder: e.g. jane.doe@company.com
+    validations:
+      required: true
+  - type: input
+    id: team-name
+    attributes:
+      label: Your Team, Role, and Company
+      description: Please provide the name of your team, your role on that team, and the name of the company you work for
+      placeholder: e.g. Platform Security, Backend Engineer, Ad Hoc
```

Comment:
What do we use this information for?

Reply:
We use the role to determine what group to put them in (`developers`, `viewers`, `platform`, etc.). For SOCKS, we don't really care about their team or company.

Reply:
👍  I'll take out the ask for their team and company and change the `id` of this input. (From what I can tell, we don't use the `id` of this section in any of our automation, so it should be okay to just change it in this one spot.)